extensions/amp-audio/0.1/amp-audio.js

```diff
@@ -63,7 +63,7 @@ export class AmpAudio extends AMP.BaseElement {
     }
     this.propagateAttributes(
         ['src', 'autoplay', 'muted', 'loop', 'aria-label',
-          'aria-describedby', 'aria-labelledby'],
+          'aria-describedby', 'aria-labelledby', 'controlsList'],
```

Comment:
Thanks @letsila!
We also need to update the validation file (`validator-amp-audio.protoascii`) and the documentation (amp-audio.md)